schemas/system/gs.domains

```diff
@@ -2,9 +2,14 @@
   Id: {
     type: 'object',
     class: 'Uint64',
-    parse: x => new api.common.Uint64(x),
-    check: id =>
-      typeof id === 'string' && new api.common.Uint64(id).toString() === id,
+    parse: x => {
```

Comment:
Something like this
```js
parse('9999999999999999999999999999999999999999999').toString()
```
will produce
```js
'7386721425538678783'
```

Reply:
So?

Reply:
Previously check compared incoming value: `new api.common.Uint64(id).toString() === id`, so I think bringing it back should help.
```js
const result = new api.common.Uint64(x);
if (result.toString() !== x.toString()) {
  throw new Error('Incorrect input value');
}
return result;
```

Reply:
This wouldn't help, it is not reliable as well, for example
```js
new Uint64('0b101010101000111').toString() !== '0b101010101000111' // true
```
Besides, there is no need to check that the input value can fully fit into Uint64 because if it can't, it will just take the lowest 64 bits of the number and use them, which is expected behavior in our case.

Reply:
Then this is fine.